client/server/bundler/sections-loader.js

```diff
@@ -53,6 +61,9 @@ const loader = function() {
 	const { forceRequire, onlyIsomorphic } = options;
 	let { include } = options;
 	let sections = require( this.resourcePath );
+	const allActiveSections = sections;
+
+	sections = allActiveSections.filter( utils.filterSections );
```

Comment:
Do we need the `allActiveSections` proxy variable? Could we just do `sections = sections.filter...`?